codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -906,9 +906,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
 
                 final boolean wasActive = isActive();
 
-                // Only ever send a reset frame if the connection is still alive as otherwise it makes no sense at
-                // all anyway.
-                if (parent().isActive() && !streamClosedWithoutError && isStreamIdValid(stream().id())) {
+                // Only ever send a reset frame if the connection is still alive and if the stream ever actually
```

Comment:
`ever actually...` -> `may have exists as otherwise we may send a RST on a stream in an invalid state and cause a connection error`. consider rewording to clarify the implications of not checking the conditions.

Reply:
done.